duckduckgo/iOS
# show privacy protection as a popover on ipad

Reviewer: Caine or Mia
Asana: https://app.asana.com/0/414235014887631/505009630947244
CC:

**Description**:

Show Privacy Protection as a popover on iPad

**Steps to test this PR**:

1. Open Privacy Protection on an iPhone. Ensure the previous behaviour remains the same.
1. Open Privacy Protection on an iPad. It should appear as a modal popover.
